src/helpers/__tests__/component-tree.test.tsx

```diff
@@ -200,3 +202,17 @@ test('returns host siblings for composite component', () => {
     view.getByTestId('siblingAfter'),
   ]);
 });
+
+test('getCompositeParentOfType', () => {
+  const view = render(<View testID="test"></View>);
+  const hostComponent = view.getByTestId('test');
+
+  const compositeComponent = getCompositeParentOfType(hostComponent, View);
+
+  // We get the corresponding composite component (same testID), but not the host
+  expect(compositeComponent?.type).toBe(View);
+  expect(compositeComponent?.props.testID).toBe(hostComponent.props.testID);
+
+  expect(compositeComponent).not.toBe(hostComponent);
+  expect(isHostElement(compositeComponent)).toBe(false);
```

Comment:
This checks seems redundant (and weaker) when we already have `expect(compositeComponent?.type).toBe(View);`